hls4ml/make_small_dataset.py

```diff
@@ -10,8 +10,17 @@
 from glob import glob
 from sklearn.utils import shuffle
 
-files_loc = "/gpfs/slac/atlas/fs1/d/rafaeltl/public/Muon/simulation/stgc/"
-# fdir = "atlas_mm_vmm_bkgr_1_TEST"
+import subprocess
```

Comment:
This is not necessary here. This file is a simple script, so one should just re-write files_loc to run somewhere else.

Reply:
If I want later to create pull request in that case the file will be pointing somewhere else for you, Is that ok?